pgx/src/trigger_support/trigger_tuple.rs

```diff
@@ -1,14 +1,13 @@
 /// Indicates which trigger tuple to convert into a [crate::PgHeapTuple].
 #[derive(Debug, Copy, Clone)]
 pub enum TriggerTuple {
-    /// The row for which the trigger was fired. This is the row being inserted, updated, or deleted.
-    /// If this trigger was fired for an INSERT or DELETE then this is what you should return from the
-    /// function if you don't want to replace the row with a different one (in the case of INSERT) or
-    /// skip the operation.
-    Current,
-
-    /// The new version of the row, if the trigger was fired for an UPDATE. This is what you have to
-    /// return from the function if the event is an UPDATE and you don't want to replace this row by
-    /// a different one or skip the operation.
+    /// Represents the new database row for INSERT/UPDATE operations in row-level triggers.
     New,
+
+    /// Represents the old database row for UPDATE/DELETE operations in row-level triggers.
+    Old,
+
+    /// Represents the old database row for UPDATE/DELETE operations or the new database row for
+    /// INSERT operations in row-level triggers.
+    Current,
```

Comment:
Hm... I don't think I understand the difference between `TriggerTuple::Current` and either of the other two variants, now. It seems there's only Current. It's not intuitive how to use the other two? If these are outputs instead of inputs, then if we are able to intuit which approach is appropriate, we shouldn't need `TriggerTuple::Current` to tell us that, we should simply _do it_.

Reply:
Hm, so I guess `New` and `Current` were intended in lieu of `New` and `Old` once upon a time?

Reply:
Yeah, and I can't tell ya why.

Reply:
`Current` is dead to us:  https://github.com/tcdi/pgx/commit/b68a29f85db91c60253fc1029b49fd0d302b2cfc